src/core/Microsoft.PowerToys.Settings.UI.Lib/BasePTModuleSettings.cs

```diff
@@ -1,15 +1,27 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
 using System.Text.Json;
 
 namespace Microsoft.PowerToys.Settings.UI.Lib
 {
     public abstract class BasePTModuleSettings
     {
+        /// <summary>
```

Comment:
Is it possible to have the stylecop set up in a way that we don't need to write these kind of comments?

Reply:
@yuyoyuppe Any reason why? We are already doing it in other parts of the [code that's in master](https://github.com/microsoft/PowerToys/blob/master/src/modules/previewpane/SvgPreviewHandler/SvgPreviewHandler.cs).

Reply:
I don't see a reason why these type of comments are needed anywhere, since they don't bring any new information, and IMO they never get properly documented.

Reply:
@udit3333 I will leave out this comment made on https://github.com/microsoft/PowerToys/pull/1933#discussion_r403425744 based on this discussion.